syntax = "proto3";

import "google/protobuf/any.proto";

package pslx;

// the next will be 7
message ProtoTableStorage {
    string table_name = 1;
    string table_path = 2;
    string created_time = 3;
    map<string, int32> key_to_row_id_mapping = 4;
    map<string, google.protobuf.Any> data = 5;
    string updated_time = 6;
}
